impl/client/src/main/javascript/web/pentaho/data/filter/isGreater.js

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright 2017 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "module",
+  "./property",
+  "./KnownFilterKind"
+], function(module, propertyFactory, KnownFilterKind) {
+
+  "use strict";
+
+  return function(context) {
+
+    var PropertyFilter = context.get(propertyFactory);
+
+    /**
+     * @name pentaho.data.filter.IsGreater.Type
+     * @class
+     * @extends pentaho.data.filter.Property.Type
+     *
+     * @classDesc The type class of the strict greater than filter type.
+     *
+     * For more information see {@link pentaho.data.filter.IsGreater}.
+     */
+
+    /**
+     * @name pentaho.data.filter.IsGreater
+     * @class
+     * @extends pentaho.data.filter.Property
+     * @amd {pentaho.type.Factory<pentaho.data.filter.IsGreater>} pentaho/data/filter/isGreater
+     *
+     * @classDesc The `IsGreater` class represents a filter for being strictly greater than a given value.
+     * The filter selects elements having the value of a certain property strictly greater than a reference
+     * value: [value]{@link pentaho.data.filter.IsGreater#value}.
+     *
+     * @description Creates a greater than filter instance.
+     *
+     * @constructor
+     * @param {pentaho.data.filter.spec.IIsGreater} [spec] - A greater than filter specification.
+     */
+
+    return PropertyFilter.extend("pentaho.data.filter.IsGreater", /** @lends pentaho.data.filter.IsGreater# */{
+
+      /** @inheritDoc */
+      get kind() {
+        return KnownFilterKind.IsGreater;
+      },
+
+      // TODO: In the future, review if value should be of type pentaho.type.Value.
+      /**
+       * Gets the expected value of the property.
+       *
+       * This getter is a shorthand for `this.getv("value")`.
+       *
+       * @type {any}
+       *
+       * @readOnly
+       */
+      get value() {
```

Comment:
Also, please remove the getter implementation, but leave a virtual doc there. The isReadOnly attribute in the declaration of the property is sufficient to achieve a similar effect (I think it still generates the setter, but throws when set).